deployments/aws/ecs/locals.tf

```diff
@@ -43,15 +43,10 @@ locals {
       TRACECAT__APP_ENV                               = var.tracecat_app_env
       TRACECAT__AUTH_ALLOWED_DOMAINS                  = var.auth_allowed_domains
       TRACECAT__AUTH_TYPES                            = var.auth_types
-      TRACECAT__SETTING_OVERRIDE_SAML_ENABLED         = var.setting_override_saml_enabled
-      TRACECAT__SETTING_OVERRIDE_OAUTH_GOOGLE_ENABLED = var.setting_override_oauth_google_enabled
-      TRACECAT__SETTING_OVERRIDE_BASIC_AUTH_ENABLED   = var.setting_override_basic_auth_enabled
```

Comment:
not sure if we should remove these, as the main reason of having TRACECAT__SETTING_OVERRIDE_XXX was to override these. If we remove these and we (hopefully don't) need this, we will have to patch these again. I'm happy to remove these from docker, because it's easier to reset or override db values easily, but for tf deploys we don't have a way to do so